coral-component-columnview/src/scripts/ColumnViewPreview.js

```diff
@@ -96,6 +96,8 @@ const ColumnViewPreview = Decorator(class extends BaseComponent(HTMLElement) {
     let element;
     let elementLabel;
 
+    Granite.I18n.get(elements);
+
```

Comment:
Coral Spectrum is a UI component library which can be used with any product, it should not contain changes related to AEM/Granite